<launch>

    <node name="scale_tools" pkg="image_processing" type="scale_manual_tools.py" output="screen">
        <!-- Parameters -->
        <param name="sub_camera_topic" value="/camera_stitched/color/image_raw/compressed" />
        <param name="pub_camera_topic" value="/camera_stitched/manual_scaled/compressed" />
        <param name="json_path" value="$(find image_processing)/config/points.json" />
    </node>

</launch>